jbmc/src/java_bytecode/ci_lazy_methods_needed.cpp

```diff
@@ -43,3 +44,114 @@ bool ci_lazy_methods_neededt::add_needed_class(
 
   return true;
 }
+
+/// Add to the needed classes all class specified, the replacement type if it
+/// will be replaced, and all feilds it contains.
```

Comment:
... `fields` ...

Reply:
Though this is a comment for #2335 so will squash into there